imaging/ml/ml_codelab/breast_density_auto_ml.ipynb

```diff
@@ -35,7 +35,7 @@
    "source": [
     "The goal of this tutorial is to train, deploy and run inference on a breast density classification model. Breast density is thought to be a factor for an increase in the risk for breast cancer. This will emphasize using the [Cloud Healthcare API](https://cloud.google.com/healthcare/) in order to store, retreive and transcode medical images (in DICOM format) in a managed and scalable way. This tutorial will focus on using [Cloud AutoML Vision](https://cloud.google.com/vision/automl/docs/beginners-guide) to scalably train and serve the model. \n",
     "\n",
-    "**Note: This is the AutoML version of the Cloud ML Engine Codelab found [here](./breast_density_cloud_ml.ipynb).**"
+    "**Note: This is the AutoML version of the Cloud ML Engine Codelab found [here](https://github.com/GoogleCloudPlatform/healthcare/blob/master/imaging/ml_codelab/breast_density_cloud_ml.ipynb).**"
```

Comment:
Please leave the relative paths for links.

Reply:
done